install.sh

```diff
@@ -81,15 +83,15 @@ do
 done
 
 tmpdir=$(mktemp -d)
-
 cd "$tmpdir"
 
-git clone -b master https://github.com/widdix/aws-ec2-ssh.git
-
-cd "$tmpdir/aws-ec2-ssh"
+AWSCLI_GITHUB_VERSION=${AWSCLI_GITHUB_VERSION:-develop}
+easy_install https://github.com/aws/aws-cli/archive/${AWSCLI_GITHUB_VERSION}.tar.gz
```

Comment:
This is not needed on Amazon linux and will confuse users as this is probably another version then the one installed with yum.
Can you make this conditional and only run it when the ```aws``` executable is not found?

Reply:
So when I was using this project I had the issue that I have multiple systems - some Ubuntu, some AmazonLinux that I want to install this project on. Having to maintain IF blocks in the CF scripts based on your current linux flavor seemed less than ideal and made the overall CF template really difficult to read. Honestly I was really frustrated that I had to go to these lengths to get the commands to run the same on both systems.

If that mindset is not a direction this project wants to go I completely understand. I would mention, however, that the cli version is configurable as I have implemented it. Is there a better default version to use? If I switched to 'master' would that be better?

My goal was that the same version of the cli would be installed on my systems whether they were AmazonLinux or Ubuntu.